woocommerce/woocommerce
# Remove id column from experimental product import sample files

### All Submissions:






### Changes proposed in this Pull Request:





Closes #36840 

This PR removes I.D column from product import sample files to prevent overwriting existing products.



### How to test the changes in this Pull Request:


1. Checkout file [A](https://github.com/woocommerce/woocommerce/blob/update/36840-remove-id-column-from-experimental_sample-file/plugins/woocommerce/sample-data/experimental_sample_9_products.csv) and [B](https://github.com/woocommerce/woocommerce/blob/update/36840-remove-id-column-from-experimental_sample-file/plugins/woocommerce/sample-data/experimental_fashion_sample_9_products.csv) on Github to make sure the format is correct.

**Testing experimental-fashion-sample-products.csv**

1. Create a new site without any products.
2. Go to product task (/wp-admin/admin.php?page=wc-admin&task=products)
3. Click `View more product types`
4. Click `Load Sample Products` and click `Import sample produdcts`
5. Go to `Products` and confirm the file has been imported.

**Testing experimental_sample_9_products.csv**

1. Create a new site without any products.
2. Turn off `experimental-fashion-sample-products` feature.
3. Go to product task (/wp-admin/admin.php?page=wc-admin&task=products)
4. Click `View more product types`
5. Click `Load Sample Products` and click `Import sample produdcts`
6. Go to `Products` and confirm the file has been imported.






### Other information:




### FOR PR REVIEWER ONLY:
